<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>volChanged</class>
 <widget class="QWidget" name="volChanged">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
background-color: rgb(245, 121, 0);
border-radius:20px

}

QToolButton:hover
{

    color:blue;
    background-color: rgb(245, 121, 0);
    border: 3px solid rgb(11 , 137 , 234);
font-size:20px;
}

QToolButton:pressed
{
    color:red;
    background-color: rgb(245, 121, 0);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:5px;
    padding-top:5px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="volDecreaseBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>减少</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="volIncreaseBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal{
width:25px;
	background-color: rgb(173, 127, 168);
margin:-11px 2px -11px 2px;
border-radius:12px;

}

QSlider::groove:horizontal{
height:15px;
background-color:green;
border-radius:7px;
padding-left:3px;
    padding-top:3px;

}
QSlider::add-page:horizontal{
background-color: rgb(52, 101, 164);
}
 QSlider::sub-page:horizontal{
background-color: rgb(52, 101, 164);
}</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="sliderPosition">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
